content/docs/getting-started/installation/kubernetes/powermax/prerequisite/_index.md

```diff
@@ -224,69 +300,74 @@ Example:
      # set password to the base64 encoded password
      password: YWRtaW4=
    ```
-These values can be obtained using base64 encoding as described in the following example:
+   These values can be obtained using base64 encoding as described in the
+   following example:
+
 ```bash
 echo -n "myusername" | base64
 echo -n "mypassword" | base64
 ```
-where *myusername* and *mypassword* are credentials for a user with vCenter privileges.
-3.
-4. Run following command to create the configmap
-  ```bash
-  kubectl create -f vcenter-secret.yaml
-  ```
->Note: Name of the secret should always be `vcenter-creds`.
 
-{{% /tab %}}
-{{< /tabpane >}}   
+where _myusername_ and _mypassword_ are credentials for a user with vCenter
+privileges. 3. 4. Run following command to create the configmap
```

Comment:
Looks like "Run following command to create the configmap" should be a separate item, probably 3.